### PR TITLE
 fix(ci): not running enough tests in parallel 

### PR DESCRIPTION
**Pull request by dpc**

### CODE REVIEW

File: devimint/src/tests.rs [hunk 1]
```diff
@@ -136,17 +137,11 @@ pub async fn latency_tests(
...
 
-    // LN operations take longer, we need less iterations
-    let iterations = 20;
-
     let cln_gw_id = gw_cln.gateway_id().await?;
 
     match r#type {
         LatencyTest::Reissue => {
             info!("Testing latency of reissue");
-            // On AlephBFT session times may lead to high latencies, so we need to run it
```
Comment (dpc) [inline devimint/src/tests.rs:147]: Not worth waiting for, plus probably requires like hundreds of iterations to really hit it.

File: scripts/tests/test-ci-all.sh [hunk 2]
```diff
@@ -237,7 +237,7 @@ fi
 if [ -n "${FM_TEST_CI_ALL_JOBS:-}" ]; then
   # when specifically set, use the env var
   parallel_args+=(--jobs "${FM_TEST_CI_ALL_JOBS}")
-elif [ -n "${CI:-}" ]; then
+elif [ -n "${CI:-}" ] || [ "${CARGO_PROFILE:-}" == "ci" ]; then
```
Comment (dpc) [inline scripts/tests/test-ci-all.sh:240]: This is the main reason. We were actually running CI with 3 jobs parallel max, while we wanted 8 (num cpus).

Comment (bradleystachurski) [reply scripts/tests/test-ci-all.sh:240]: Interesting, I would have expected `CI=true` from the GHA docs

https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

**Review by bradleystachurski — APPROVED** [review]

File: scripts/tests/test-ci-all.sh (hunk 2 above)
Comment (dpc) [reply scripts/tests/test-ci-all.sh:240]: That was the mistake. It runs in nix derivation so CI envvar is not set! 😂

Comment (dpc) [reply scripts/tests/test-ci-all.sh:240]: I probably assumed the same thing and didn't notice.

Comment (bradleystachurski) [reply scripts/tests/test-ci-all.sh:240]: Ah! Nice find 😄

**Pull request merged**